salt/api-gateway/config/etc-kong-kong.conf

```diff
@@ -54,57 +54,53 @@ anonymous_reports = off          # Send anonymous usage data such as error
 
 
 # modified
-proxy_listen = 0.0.0.0:80     # Address and port on which Kong will accept
+#proxy_listen = 0.0.0.0:80       # Address and port on which Kong will accept
```

Comment:
I hope the forwarding works first time because without this the end2end environment will stop working if it's not correctly picked up by the nginx instance in front. But this should be testable inside Vagrant so we should be sure that someone is listening on port 80